docs/SUT-VERSIONS.md

```diff
@@ -56,4 +56,18 @@ as expected (and it is also a good way to speed up your tests since no slow plug
 executed during your tests).
 
 Note that this option is not yet supported by all [Jenkins controllers](CONTROLLER.md), 
-so e.g. use `TYPE=winstone` in order to get the `PLUGINS_DIR` option working.
\ No newline at end of file
+so e.g. use `TYPE=winstone` in order to get the `PLUGINS_DIR` option working.
+
+### Running in pre configured plugin mode
```

Comment:
I suggest to squash this to previous section covering both `PLUGIN_DIR` and the bundle use-case.